content/docs/kitty/building.md

```diff
@@ -7,15 +7,18 @@ draft = false
 # Building the Kitty system
 
 ## Acquire source code
-(this takes around 1.7Gb to download)
+
+Acquiring the source code involves initialising several submodules
+that are found in the `dep/` directory. We can initialise these through
+a build step in the Kitty Makefile, called `submodules`, which will only
+initialise the submodules that are required for this example system.
+This takes around 1.7 GB to download.
 
 ```sh
-git clone https://github.com/au-ts/lionsos.git
-cd lionsos
-git submodule update --init
-cd micropython
-git submodule update --init lib/micropython-lib
-cd ..
+cd examples/kitty
+export MICROKIT_SDK=/path/to/sdk
```

Comment:
This export is below so is not necessary here.

Reply:
also at this point they haven't acquired the SDK so this is not possible

Reply:
I imagine this is because the makefile throws an error if you do not pass `MICROKIT_SDK`? we should fix that first I guess.

Reply:
Yeh thats correct, which is one of the reasons why i had a separate build step initially for it. I'll look into fixing that. For now should we just move the submodule init section to the bottom?

Reply:
Hmm

Reply:
yes

Reply:
I'll fix it now.